pkg/cache/context_test.go

```diff
@@ -2433,3 +2433,111 @@ func assertListerPods(pods []*v1.Pod, count int) bool {
 	}
 	return count == counted
 }
+
+type MockRetryStrategy struct {
+	totalSleep time.Duration
+}
+
+func (m *MockRetryStrategy) Sleep(duration time.Duration) {
+	m.totalSleep += duration
+}
```

Comment:
Extra code, not needed

Reply:
I added this to expose to testing code to mock the sleep time interval, we don't have a the context mock class.

Reply:
Addressed the new API in latest PR, also removed the extra code.